libraries/chain/controller.cpp

```diff
@@ -2293,7 +2293,7 @@ struct controller_impl {
          ilog("switching forks from ${current_head_id} (block number ${current_head_num}) to ${new_head_id} (block number ${new_head_num})",
               ("current_head_id", head->id)("current_head_num", head->block_num)("new_head_id", new_head->id)("new_head_num", new_head->block_num) );
 
-         if (auto dm_logger = get_deep_mind_logger()) {
+         if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
This is inside `maybe_switch_forks`, which is not transaction specific.